src/postamble.js

```diff
@@ -262,6 +266,41 @@ function run(args) {
   stackCheckInit();
 #endif
 
+#if RELOCATABLE
+  if (!dylibsLoaded) {
+  // Loading of dynamic libraries needs to happen on each thread, so we can't
+  // use the normal __ATPRERUN__ mechanism.
+#if MAIN_MODULE
+    preloadDylibs();
+#else
+    reportUndefinedSymbols();
```

Comment:
Not related to this PR, as it's pre-existing, but do you know why the main thread loads dylibs while the side module reports undefined symbols? That seems odd. I'd expect the main module to do both?

Reply:
Oh, wait, the side module has no JS. So basically, if it's relocatable but not a main module, it only reports undefined symbols, I guess?

Reply:
Indeed.  Although I'm not really sure how much that configuration gets used .. I don't think we test it much.